tests/test_glue/test_parser.sphinx4.xml

```diff
@@ -0,0 +1,169 @@
+<document source="with_glue">
```

Comment:
Should we now expect a new one of these files for every major version of Sphinx that we support?

Reply:
@choldgraf the `sphinx3` is the same as the old fixture file. There is a minor difference in what sphinx includes in the xml "align='default`" is no longer specified in `sphinx4`